<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>628</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>YADL v0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Launcher</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="tab1_layout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <property name="spacing">
           <number>10</number>
          </property>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Files</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QCheckBox" name="tab1_ch_coop">
            <property name="text">
             <string>Co-op enemies</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="tab1_cb_iwad"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>IWAD</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Additional params</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QComboBox" name="tab1_cb_map"/>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QComboBox" name="tab1_cb_complevel">
            <item>
             <property name="text">
              <string>Default or N/A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2 - Doom v1.9 (vanilla Doom and Doom 2)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3 - Ultimate Doom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4 - Final Doom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9 - Boom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11 - MBF</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>21 - MBF21</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QComboBox" name="tab1_cb_port"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Skill</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Comp. level</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Map</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QComboBox" name="tab1_cb_skill">
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1 - I'm Too Young To Die</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2 - Hey, Not Too Rough</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3 - Hurt Me Plenty</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4 - Ultra-Violence</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5 - Nightmare!</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <layout class="QGridLayout" name="tab1_layout_files">
            <item row="1" column="0">
             <widget class="QPushButton" name="tab1_bt_add">
              <property name="toolTip">
               <string>Add file</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../assets/assets.qrc">
                <normaloff>:/icons/icons/icon_add.xpm</normaloff>:/icons/icons/icon_add.xpm</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="tab1_bt_up">
              <property name="toolTip">
               <string>Move file up</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../assets/assets.qrc">
                <normaloff>:/icons/icons/icon_up.xpm</normaloff>:/icons/icons/icon_up.xpm</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="tab1_bt_down">
              <property name="toolTip">
               <string>Move file down</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../assets/assets.qrc">
                <normaloff>:/icons/icons/icon_down.xpm</normaloff>:/icons/icons/icon_down.xpm</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="tab1_bt_remove">
              <property name="toolTip">
               <string>Remove file</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../assets/assets.qrc">
                <normaloff>:/icons/icons/icon_delete.xpm</normaloff>:/icons/icons/icon_delete.xpm</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="4">
             <widget class="QListView" name="tab1_list_files"/>
            </item>
           </layout>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QLineEdit" name="tab1_entry_params"/>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="tab1_ch_fast">
            <property name="text">
             <string>Fast monsters</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="tab1_bt_preset">
          <property name="text">
           <string>Save preset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tab1_bt_launch">
          <property name="text">
           <string>Launch game</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Ports/IWADs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="tab2_layout_ports">
          <item row="2" column="2">
           <widget class="QPushButton" name="tab2_bt_portdel">
            <property name="toolTip">
             <string>Remove sourceport</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_delete.xpm</normaloff>:/icons/icons/icon_delete.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="tab2_bt_portadd">
            <property name="toolTip">
             <string>Add source port</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_add.xpm</normaloff>:/icons/icons/icon_add.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="tab2_bt_portup">
            <property name="toolTip">
             <string>Move sourceport up</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_up.xpm</normaloff>:/icons/icons/icon_up.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="tab2_bt_portdown">
            <property name="toolTip">
             <string>Move sourceport down</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_down.xpm</normaloff>:/icons/icons/icon_down.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="tab2_bt_portedit">
            <property name="toolTip">
             <string>Edit source port</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_edit.xpm</normaloff>:/icons/icons/icon_edit.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="5">
           <widget class="QTableView" name="tab2_table_ports">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="tabKeyNavigation">
             <bool>true</bool>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QLabel" name="tab2_label_ports">
            <property name="text">
             <string>Source ports</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="tab2_layout_iwads">
          <item row="2" column="3">
           <widget class="QPushButton" name="tab2_bt_iwadup">
            <property name="toolTip">
             <string>Move IWAD up</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_up.xpm</normaloff>:/icons/icons/icon_up.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="tab2_bt_iwadadd">
            <property name="toolTip">
             <string>Add IWAD</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_add.xpm</normaloff>:/icons/icons/icon_add.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="tab2_bt_iwaddel">
            <property name="toolTip">
             <string>Remove IWAD</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_delete.xpm</normaloff>:/icons/icons/icon_delete.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="tab2_bt_iwaddown">
            <property name="toolTip">
             <string>Move IWAD down</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_down.xpm</normaloff>:/icons/icons/icon_down.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="tab2_bt_iwadedit">
            <property name="toolTip">
             <string>Edit IWAD</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_edit.xpm</normaloff>:/icons/icons/icon_edit.xpm</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="5">
           <widget class="QTableView" name="tab2_table_iwads">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QLabel" name="tab2_label_iwads">
            <property name="text">
             <string>IWADs</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>Presets</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="tab3_layout">
          <item row="0" column="1">
           <spacer name="tab3_spacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="tab3_bt_load">
            <property name="toolTip">
             <string>Load preset</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_play.xpm</normaloff>:/icons/icons/icon_play.xpm</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="tab3_bt_down">
            <property name="toolTip">
             <string>Move preset down</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_down.xpm</normaloff>:/icons/icons/icon_down.xpm</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="tab3_bt_up">
            <property name="toolTip">
             <string>Move preset up</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_up.xpm</normaloff>:/icons/icons/icon_up.xpm</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="tab3_bt_remove">
            <property name="toolTip">
             <string>Remove preset</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../assets/assets.qrc">
              <normaloff>:/icons/icons/icon_delete.xpm</normaloff>:/icons/icons/icon_delete.xpm</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="6">
           <layout class="QVBoxLayout" name="tab3_layout_table">
            <item>
             <widget class="QLabel" name="tab3_label_presets">
              <property name="text">
               <string>Saved presets</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tab3_table_presets">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="tab3_entry_filter">
              <property name="placeholderText">
               <string>Filter presets...</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
